convergence_elastic/parameters2.par

```diff
@@ -0,0 +1,61 @@
+&Equations
+rho = 1.
+mu = 1.

```

Comment:
What's the purpose of this file? Did you commit this by accident?

Reply:
probably yes

Reply:
ok that should be fixed now